images/ubuntu/templates/ubuntu-24.04.pkr.hcl

```diff
@@ -258,11 +263,11 @@ build {
     scripts          = ["${path.root}/../scripts/build/install-apt-vital.sh"]
   }
 
-provisioner "shell" {
-    environment_vars = ["HELPER_SCRIPTS=${var.helper_script_folder}", "INSTALLER_SCRIPT_FOLDER=${var.installer_script_folder}"]
-    execute_command  = "sudo sh -c '{{ .Vars }} {{ .Path }}'"
-    scripts          = ["${path.root}/../scripts/build/install-powershell.sh"]
-  }
+  provisioner "shell" {
+      environment_vars = ["HELPER_SCRIPTS=${var.helper_script_folder}", "INSTALLER_SCRIPT_FOLDER=${var.installer_script_folder}"]
+      execute_command  = "sudo sh -c '{{ .Vars }} {{ .Path }}'"
+      scripts          = ["${path.root}/../scripts/build/install-powershell.sh"]
+    }
```

Comment:
Why we need double spaces in this section? Does this change anything?

Reply:
Earlier this snippet has some indentation issue, i forgot to remove those spaces. i will rectify them. Thankyou @Alexey-Ayupov 

will create another PR by taking your inputs.